### ISSUE TITLE
att_masks

### ISSUE THREAD

**Issue by gorgeousyouth** [ruotianluo/DiscCaptioning#12]
你好，大佬，不知道你有没有空，想问你个问题，请问一下你发布的imagecaption中的dataload中加载的att_mask作用是什么？

**Comment by ajfisch** [comment]
I would also like to know this. What work has to be done to get dataloaderraw to work? I would like to run inference on raw images.

**Comment by sabirdvd** [comment]
> I would also like to know this. What work has to be done to get dataloaderraw to work? I would like to run inference on raw images.

I found this  link that solve the mask problem  
https://github.com/ruotianluo/self-critical.pytorch/issues/42
